test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -50,11 +74,37 @@ DEFINE_PROTO_FUZZER(Contract const& _input)
 		// We always call the function test() that is defined in proto converter template
 		hexEncodedInput = methodIdentifiers["test()"].asString();
 	}
-	catch (...)
+	catch (Exception const&)
 	{
-		cout << contract_source << endl;
-		throw;
+		return;
 	}
-	// TODO: Call evmone wrapper here
-	return;
+
+	// We target the default EVM which is the latest
```

Comment:
It may be better to create a special function for everything that follows here and then also have the VM be a static variable inside that function.